django/test/runner.py

```diff
@@ -708,6 +713,7 @@ def run_tests(self, test_labels, extra_tests=None, **kwargs):
         run_failed = False
         try:
             self.run_checks(databases)
+            self.clone_databases(old_config)
```

Comment:
Shouldn't this be called from `setup_databases`?

Reply:
Per [this discussion](https://github.com/django/django/pull/12646#discussion_r455179445), database cloning has been split up to its own method call